src/components/Alteration/Articles/ResolutionDates.vue

```diff
@@ -148,7 +148,7 @@ export default class ResolutionDates extends Mixins(CommonMixin, DateMixin) {
   @Getter haveNewResolutionDates!: boolean
   @Getter getIsResolutionDatesValid!: boolean
   @Getter isSummaryMode!: boolean
-  @Getter isBenCorrectionFiling!: boolean
+  @Getter isCorrectionFiling!: boolean
```

Comment:
This code applies to all corrections, not just BEN corrections. (Looking at it another way, corrections that don't need this component don't use it.)